Client/core/CCore.cpp

```diff
@@ -26,6 +26,7 @@
 #include <SharedUtil.Detours.h>
 #include <ServerBrowser/CServerCache.h>
 #include "CDiscordRichPresence.h"
+#include <curl/include/curl/curl.h>
```

Comment:
Add "../../vendor/curl/include" to "includedirs" in  premake5.lua for Client Core.  This allows you to use "#include <curl/curl.h>" here.